src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function add(...args) {
```

Comment:
The add function is using rest parameters, which is unnecessary since the `adder` function should only take one argument at a time. Consider simplifying the implementation by expecting a single argument.